### DISCUSSION TITLE
Missing frozen string literal comment

### DISCUSSION THREAD

**Discussion by kaihendry** [Homebrew/discussions#512] [Everyday usage]
Hi guys, I'm stuck at the audit stage here:

```
(ins)hendry-tw-mbp~/brewery$ brew audit --strict --new-formula --online goredo.rb
goredo:
  * 1: col 1: No Sorbet sigil found in file. Try a `typed: false` to start (you can also use `rubocop -a` to automatically add this).
  * 1: col 1: Missing frozen string literal comment.
Error: 2 problems in 1 formula detected
(ins)hendry-tw-mbp~/brewery$ rubocop
-bash: rubocop: command not found
(ins)hendry-tw-mbp~/brewery$ cat goredo.rb
# https://aur.archlinux.org/packages/goredo/
class Goredo < Formula
  desc "Go implementation of djb's redo, a Makefile replacement that sucks less"
  homepage "http://www.goredo.cypherpunks.ru/"
  url "http://www.goredo.cypherpunks.ru/download/goredo-0.11.0.tar.zst"
  version "0.11.0"
  sha256 "4608b75bc8a3dfb62ef305b5eed0f4b28ad39f64ae46baceea8100d90a8666f9"
  license "GPL-3.0-or-later"

  depends_on "go" => :build
  depends_on "zstd" => :build

  def install
    goredo_prefix = "goredo-#{version}"
    system "tar", "--use-compress-program", "unzstd", "-xvf", "#{goredo_prefix}.tar.zst"
    cd goredo_prefix do
      ENV["GOPATH"] = Pathname.pwd
      system "go", "build", *std_go_args, "go.cypherpunks.ru/goredo"
    end
    cd bin do
      system "./goredo", "-symlinks"
    end
  end

  test do
    assert_match version.to_s, shell_output("#{bin}/goredo-ifchange -v")
  end
end
```
Surprised that the audit lint messages don't appear to have a reference online, so that you can figure out how to fix them!

Can anyone help and please review **my first formula** before I create a PR?

I worked from https://github.com/Homebrew/homebrew-core/pull/68732 which has the same issue.

**Comment by gromgit** [comment]
Assuming you're planning to submit to `homebrew/core`, suggest you move your formula into the `$(brew --repo homebrew/core)/Formula` directory and `brew audit` again, this time without the `.rb` extension.

**Comment by kaihendry** [comment]
Thanks!

```
(ins)hendry-tw-mbp/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula$ cp ~/brewery/goredo.rb .
(ins)hendry-tw-mbp/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula$ brew audit --strict --new-formula --online goredo
(ins)hendry-tw-mbp/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula$
```

Now do I have to fork the repo in order to make a PR, it's been awhile...

Just created https://github.com/Homebrew/homebrew-core/pull/68846 via the Github UI. Quite straightforward I must concede.

**Reply by gromgit** [reply]
Have you read https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request? For starters, there's a naming convention for new formula PRs, like the one you originally pointed to.

**Reply by kaihendry** [reply]
Thanks Adrian, I didn't know that. I updated the PR title to hopefully meet the convention. I will do the git setup shortly.
